.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+name: Tests
+jobs:
+  tests:
+    name: ${{ matrix.target }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        target: [runtime-test]
+    runs-on: ${{ matrix.os }}
+    env:
+      RUST_TOOLCHAIN: "nightly-2021-03-15"
```

Comment:
And use `nightly` here?

Reply:
we cant use nightly since srtool latest image is above